sklearn/datasets/_openml.py

```diff
@@ -721,6 +726,11 @@ def fetch_openml(name=None, *, version='active', data_id=None, data_home=None,
         in 'target' are represented as NaN's (numerical target) or None
         (categorical target)
     """
+    if as_frame == 'warn':
+        warn("The default value of as_frame will change from False "
```

Comment:
I think there should be something to say how to get rid of the warning. Please use as_frame=False to get rid of this warning.

Reply:
Probably you want to complement with the second option `or 'auto' to get a DataFrame when it is possible`

Reply:
updated the warning message. The message does not ask the user to use any specific option (`False`, `True` or `'auto'`) because I don't want to make any assumption about what the user wants to achieve. They can refer to docstring for details.

Reply:
Fine with me.